operations.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import json
+import subprocess
+import os
+
+from connectors.core.connector import Connector, get_logger, ConnectorError
+
+logger = get_logger('dnstwist')
+DNS_TWIST = "/opt/cyops-integrations/.env/bin/dnstwist"
+
+
+def search(config, params):
+    domain = params.get("domain").strip()
+    cli_params = [DNS_TWIST,
+                  "--registered",
+                  "-f",
+                  "json",
+                  f"{domain}"]
+    try:
+
+        result = subprocess.run(cli_params, stdout=subprocess.PIPE)
```

Comment:
using 'import dnstwist' then DomainFuzz() it's probably a better pythonic way to implement the search action rather than subprocess

Reply:
subprocess canceled, changed to a function call but it is more complex now. solved